qemu/cpus.c

```diff
@@ -191,15 +169,10 @@ static int qemu_tcg_init_vcpu(CPUState *cpu)
         uc->tcg_halt_cond = cpu->halt_cond;
         snprintf(thread_name, VCPU_THREAD_NAME_SIZE, "CPU %d/TCG",
                 cpu->cpu_index);
-        if (qemu_thread_create(uc, cpu->thread, thread_name, qemu_tcg_cpu_thread_fn,
-                cpu, QEMU_THREAD_JOINABLE))
-            return -1;
+        qemu_thread_get_self(uc, cpu->thread);
```

Comment:
Which raises the question, is unicorn ever intended to emulate a multi-cpu system? Currently num_cpus and num_threads is hard coded to 1 each I believe.